core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -36,6 +37,39 @@
 public class CustomExceptionHandler extends ResponseEntityExceptionHandler {
     private ErrorAttributes errorAttributes;
 
+    /**
+     * Method intercept exception {@link UserHasNoAvailableHabitDictionaryException}.
+     *
+     * @param ex      Exception witch should be intercepted.
+     * @param request contain  detail about occur exception
+     * @return ResponseEntity witch  contain http status and body  with message of exception.
+     * @author Yurii Savchenko
+     */
+    @ExceptionHandler(UserHasNoAvailableHabitDictionaryException.class)
+    public final ResponseEntity<Object> handleUserHasNoAvailableHabitDictionaryException(
+        UserHasNoAvailableHabitDictionaryException ex, WebRequest request) {
+        log.info(ex.getMessage());
+        ExceptionResponse exceptionResponse = new ExceptionResponse(getErrorAttributes(request));
+        return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(exceptionResponse);
+    }
+
+    /**
+     * Method intercept exception {@link ConstraintViolationException}.
+     *
+     * @param ex      Exception witch should be intercepted.
+     * @param request contain  detail about occur exception
+     * @return ResponseEntity witch  contain http status and body  with message of exception.
+     * @author Yurii Savchenko
+     */
+    @ExceptionHandler(ConstraintViolationException.class)
+    public final ResponseEntity<Object> handleConstraintViolationException(ConstraintViolationException ex,
```

Comment:
Now there is a new problem. `ConstraintViolationException` is a generic exception that is thrown on any validation error. So whatever is the reason user will receive `SELECT_CORRECT_LANGUAGE` message. 
Can you send instead the message extracted from `ex` object instead? 
Could you please check that my suggestions are correct?